schema/queue/configuration/QueueConfiguration.go

```diff
@@ -179,3 +185,34 @@ func (mrc *MessageRouterConfiguration) FindQueuesByAttr(attrs []string) map[stri
 	mrc.Logger.Debug().Msg("Queue was found")
 	return result
 }
+
+func (qc *QueueConfig) Verify(messages *p_buff.MessageGroupBatch) bool {
```

Comment:
We shouldn't implement the interface method on the configuration object. We should pass the configuration into another struc and implement the interface methods for that struc

Reply:
Here I was trying to do it based on Nikita's comment above as he said 
_"We talk about avoiding new object creation during filter.
We can add
type Filter interface {
func Verify(messages *p_buff.MessageGroupBatch)
}
and develop an implementation which one gets required value from MessageGroupBatch and compare them with internal expected value by internal operation."_    
here only way don't to pass anything to "Verify" except messages, thought was to implement it for configuration, also tried to avoid creation of object on which he said that. Seems like bad solution, will return that part as it was